pkg/controller/lambda/function/setup.go

```diff
@@ -271,18 +272,48 @@ type updater struct {
 	client svcsdkapi.LambdaAPI
 }
 
+func (u *updater) isLastUpdateStatusSuccessful(cr *svcapitypes.Function) error {
+	for {
```

Comment:
Should I put some kind of limit on this? How long do we want to wait for the status?